### PR TITLE
Add separate enrichers page and combine policies pages

### PR DESCRIPTION
**Pull request by mikezaccardo**
This implements a structural change that I proposed in a previous PR and was supported[1] by @geomacy.

[1] https://github.com/apache/brooklyn-docs/pull/159#pullrequestreview-29928236

### CODE REVIEW

File: guide/java/enrichers.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+{% endhighlight %}
+
+#### Propagator
+
+- org.apache.brooklyn.enricher.stock.Propagator
+
+Use propagator to duplicate one sensor as another, giving the supplied sensor mapping.
+The other use of Propagator is where you specify a producer (using `$brooklyn:entity(...)` as below)
+from which to take sensors; in that mode you can specify `propagate` as a list of sensors whose names are unchanged,
+instead of (or in addition to) this map
```
Comment (drigodwin) [inline guide/java/enrichers.md]: Missing `.`

File: guide/java/enrichers.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    enricher.targetSensor: $brooklyn:sensor("webapp.reqs.perSec.perNode")
+    enricher.aggregating.fromMembers: true
+    transformation: average
+{% endhighlight %}
+
+#### Joiner
+
+- org.apache.brooklyn.enricher.stock.Joiner
+
+Joins a sensor whose output is a list into a single item joined by a separator.
```
Comment (drigodwin) [inline guide/java/enrichers.md]: What's the default seperator and can you change this?

File: guide/java/enrichers.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    enricher.sourceSensor: $brooklyn:sensor("urls.tcp.list")
+    enricher.targetSensor: $brooklyn:sensor("urls.tcp.string")
+    uniqueTag: urls.quoted.string
+{% endhighlight %}
+
+####	Delta Enricher
+
+- org.apache.brooklyn.policy.enricher.Delta Enricher
+
+Converts absolute sensor values into a delta.
```
Comment (drigodwin) [inline guide/java/enrichers.md]: What's does delta mean in this context? perhaps include an example.

File: guide/java/enrichers.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+    enricher.sourceSensor: reqs.count
+    enricher.targetSensor: reqs.per_sec
+    enricher.delta.period: 1s
+{% endhighlight %}
+
+####	Rolling Mean
+
+- org.apache.brooklyn.policy.enricher.RollingMeanEnricher
+
+Converts the last *N* sensor values into a mean.
```
Comment (drigodwin) [inline guide/java/enrichers.md]: Is N a config value to the enricher? What's it's key?

File: guide/java/enrichers.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+- org.apache.brooklyn.policy.enricher.RollingTimeWindowMeanEnricher.HttpLatencyDetector
+
+An Enricher which computes latency in accessing a URL.
+
+#### Combiner
+
+- org.apache.brooklyn.enricher.stock.Combiner
+
+Can be used to combine the values of sensors.  This enricher should be instantiated using Enrichers.buider.combining(..).
```
Comment (drigodwin) [inline guide/java/enrichers.md]: Put backticks round this?

File: guide/java/policies.md [hunk 6]
```diff
@@ -121,151 +120,79 @@ The ConditionalSuspendPolicy will suspend and resume a target policy based on co
...
-    uniqueTag: urls.quoted.string
+Policies perform the active management enabled by Brooklyn.
+Each policy instance is associated with an entity,
+and at runtime it will typically subscribe to sensors on that entity or children,
+performing some computation and optionally actions when a subscribed sensor event occurs.
+This action might be invoking an effector or emitting a new sensor,
+depending the desired behavior is.
+
+Writing a policy is straightforward.
+Simply extend ``AbstractPolicy``,
```
Comment (drigodwin) [inline guide/java/policies.md]: Should this and below have double backticks?

File: guide/java/policies.md [hunk 7]
```diff
@@ -121,151 +120,79 @@ The ConditionalSuspendPolicy will suspend and resume a target policy based on co
...
-(assuming the `load.balancer.entity` sensor returns a Brooklyn entity), this will not function properly because `enricher.producer`
-will unsuccessfully attempt to get the supplied entity immediately.
+Order policies carefully, and mark sensors as "handled" (or potentially "swallow" them locally), so that subsequent policies and parent entities do not take superfluous (or contradictory) corrective action.
 
-Next: Writing a Policy
----------------------------
+### Implementation Classes
 
-To write a policy, see the section on [Writing a Policy]({{ site.path.guide }}/java/policy.html).
+- extend ``AbstractPolicy``, or override an existing policy
```
Comment (drigodwin) [inline guide/java/policies.md]: Perhaps link to the javadoc or code

File: guide/java/enrichers.md (hunk 4 above)
Comment (mikezaccardo) [inline guide/java/enrichers.md]: TODO

File: guide/java/enrichers.md (hunk 3 above)
Comment (mikezaccardo) [inline guide/java/enrichers.md]: TODO

File: guide/java/enrichers.md (hunk 2 above)
Comment (mikezaccardo) [inline guide/java/enrichers.md]: TODO

File: guide/java/enrichers.md [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+categories: [use, guide, defining-applications]
+---
+
+Enrichers provide advanced manipulation of an entity's sensor values.
+
+#### Transformer
+
+- org.apache.brooklyn.enricher.stock.Transformer
+
+Transforms attributes of an entity.
```
Comment (mikezaccardo) [inline guide/java/enrichers.md]: TODO

File: guide/java/enrichers.md [hunk 9]
```diff
@@ -0,0 +1,150 @@
+---
+title: Enrichers
+layout: website-normal
+toc: ../guide_toc.json
+categories: [use, guide, defining-applications]
+---
+
+Enrichers provide advanced manipulation of an entity's sensor values.
```
Comment (mikezaccardo) [inline guide/java/enrichers.md]: TODO

**Pull request merged**